server/build.gradle

```diff
@@ -30,6 +30,10 @@
 
 import org.opensearch.gradle.info.BuildParams
 
+plugins {
+  id('com.google.protobuf') version '0.8.19'
```

Comment:
`com.google.protobuf` has `0.9.2` latest version. Also, can we add the version in version.properties and fetch it from there?